site-src/guides/multiple-ns.md

```diff
@@ -1,38 +1,65 @@
+
+In this example, there are two independent teams, store and site, operating in the same Kubernetes cluster, in the `store` and `site` Namespaces. These are their requirements:
+
+- The site team has two applications, home and login, that are running behind `foo.example.com`. They have decided to deploy independent HTTPRoutes for each application so that the routing can be configured indendently for each app. By having an independent HTTPRoute for each app, it allows them to update the canary weights for the login app without having to touch any routing configuration for home.
+- The store team has a single Service called `store` that they have deployed in the `store` Namespace. 
+- The Foobar Corporation operates behind the `foo.example.com` domain so they would like to host all applications on the same Gateway resource. This is controlled by a central infrastructure team, operating in the `infra` Namespace.
+- Lastly, the security team controls the certificate for `foo.example.com`. By managing this certificate through the single shared Gateway they are able to centrally control security without directly involving application teams.
+
+The logical relationship between the Gateway API resources looks like this:
+
+![Cross-Namespace routing](/images/cross-namespace-routing.svg)
+
+## Cross-namespace Route binding
+
+[Route binding](/concepts/api-overview/#route-binding) is an important concept that dictates how routes and Gateways select eachother for binding. It is especially relevant when there are multiple Gateways and multiple Namespaces in a cluster. Route binding is based on the principle of bi-directional selection - this means that each resource, the Gateway and the Route, have independent control to determine which resources they permit binding with. This allows Route-owners and Gateway-owners to be independent actors. Route-owners can specify that they will bind with all Gateways in the cluster, or only Gateways from a specific Namespace, with a specific label selector, or an individual Gateway. Similarly, Gateways provide the same level of control. This allows a cluster to be more self-governed, which requires less central administration to ensure that Routes are not over-exposed.
+
+## Resource Deployment
+
+The infrastructure team deploys the `shared-gateway` Gateway into the `infra` Namespace. 
+
+```yaml
+{% include 'cross-namespace-routing/gateway.yaml' %}
 ```
-{% include 'routes-in-multiple-namespaces.yaml' %}
+
+A couple notes about this Gateway:
+
+- It is matching for the `foo.example.com` domain. This is configured on the Gateway so that each HTTPRoute does not also have to configure hostname matching, since they are all using the same domain.
```

Comment:
some addition: This also allows for using the same HTTPRoute resources across development and production environments where the development environment might be hosted at `foo.dev.corp.example.com`. (dev/prod have separate clusters).

Reply:
Good point, I'll add it in